### PR TITLE
add interval field test in series query

### PR DESCRIPTION
**Pull request by Sild**

### CODE REVIEW

**Comment by GregoryKutuzov** [comment]
ok

File: src/test/java/com/axibase/tsd/api/method/series/SeriesQueryTest.java [hunk 1]
```diff
@@ -26,6 +29,7 @@
     private static final String sampleDate = "2016-07-01T14:23:20.000Z";
     private static final Series series;
     private static Calendar calendar = Calendar.getInstance();
+    private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```
Comment (rmakulov) [inline src/test/java/com/axibase/tsd/api/method/series/SeriesQueryTest.java]: Why we can't use here explicit SeriesQueryTest.class?

Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/series/SeriesQueryTest.java]: That's more common solution, we can just take this definition to another class without any changes. And be sure that logs will be written by right class.

**Comment by rmakulov** [comment]
ok

**Pull request merged**